OpenAI-DotNet/OpenAIClient.cs

```diff
@@ -209,5 +213,16 @@ private HttpClient SetupClient(HttpClient client = null)
         /// <see href="https://platform.openai.com/docs/api-reference/assistants"/>
         /// </summary>
         public AssistantsEndpoint AssistantsEndpoint { get; }
+
+        /// Create messages within threads
+        /// <see href="https://platform.openai.com/docs/api-reference/messages"/>
+        /// </summary>
+        public ThreadMessagesEndpoint ThreadMessagesEndpoint { get; }
+
+        /// <summary>
+        /// Represents an execution run on a thread.
+        /// <see href="https://platform.openai.com/docs/api-reference/runs"/>
+        /// </summary>
+        public ThreadRunsEndpoint ThreadRunsEndpoint { get; }
```

Comment:
We don't need separate endpionts for these, lets combine them in `ThreadsEndpoint`